assets/src/js/admin/onboarding-wizard/components/donation-form/index.js

```diff
@@ -1,20 +1,36 @@
+// Import vendor dependencies
+import { useState } from 'react';
+const { __ } = wp.i18n;
+
+// Import utilities
+import { getWindowData } from '../../utils';
+
+// Import components
+import ConfigurationIcon from '../icons/configuration';
+
+// Import styles
+import './style.scss';
+
 const DonationForm = () => {
-	const placeholderStyle = {
-		boxShadow: '0px 1.74499px 4.36246px rgba(0, 0, 0, 0.305862)',
-		borderRadius: '6px',
-		width: '480px',
-		height: '595px',
-		background: '#fff',
-		display: 'flex',
-		alignItems: 'center',
-		justifyContent: 'center',
-		fontSize: '24px',
-		lineHeight: '32px',
-		textAlign: 'center',
+	const formPreviewUrl = getWindowData( 'formPreviewUrl' );
+	const [ iframeLoaded, setIframeLoaded ] = useState( false );
+
+	const iframeStyle = {
+		opacity: iframeLoaded === false ? '0' : '1',
```

Comment:
Is there a particular reason that you are toggling between the elements by swapping the opacity, as opposed to conditionally rendering the elements?

Reply:
In this case it's just a visual thing. Wanted to avoid jarring transitions between these states, and the fade between states is already present in the Multi-Step form template, so it felt right to mirror that here.